pkg/db/interface.go

```diff
@@ -362,11 +379,13 @@ func (d *dbConn) getTrials(trialID string, studyID string) ([]*api.Trial, error)
 		trial := new(api.Trial)
 
 		var parameters, tags string
+		var timeStamp string
 		err := rows.Scan(&trial.TrialId,
 			&trial.StudyId,
 			&parameters,
 			&trial.ObjectiveValue,
 			&tags,
+			&timeStamp,
```

Comment:
In other grpc APIs, `RFC3339Nano` format is used, not MySQL format.
Please convert timeStamp to the format when returning api.Trial.

Reply:
Right now, in the Trial API we don't have time field.
https://github.com/kubeflow/katib/blob/master/pkg/api/api.proto#L363
We store it in database, but can't get with grpc APIs.
Should we extend this API with time and add parsing in the separate PR?

Reply:
I see. 
Yes, please add a timestamp field to grpc API and write doc about the format.

Reply:
Ok, I will do it in a separate PR.